### DISCUSSION TITLE
Release v1.15.4 / 5.70.4

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#4438] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

### Release Notes

This build resolves many issues including multiple system stability issues.

For a full list of changes please review the [change log](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1154--5704---2024-12-19).

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.15.4'>Release v1.15.4 / 5.70.4</a>.</em>

**Comment by love-code-yeyixiao** [comment]
So maybe we could add a troubling wiazrd for the firefox issue?

**Comment by enrique-henri** [comment]
Might just be me but I just noticed that my certificate shows as expired 30 days ago, this is in error.
Renewed Patreon annual subs September 2024.

**Comment by Rdxsingh7** [comment]
pro aare here
